README.md

```diff
@@ -128,95 +130,145 @@ colcon build --cmake-args -DCMAKE_BUILD_TYPE=RELEASE
 
 ### Barebones Example
 
-This example emulates a running robot and also a running free fleet server,
+This example emulates a running ROS 1 robot,
 
 ```bash
-source ~/client_ws/install/setup.bash
+source ~/ff_ros1_ws/install/setup.bash
 roslaunch ff_examples_ros1 fake_client.launch
 ```
 
-The client will then start subscribing to all the necessary topics, and start publishing robot states over DDS to the server. Start the server using
+This example emulates a running ROS 2 robot,
 
 ```bash
-source ~/server_ws/install/setup.bash
+source ~/ff_ros2_ws/install/setup.bash
+ros2 launch ff_examples_ros2 fake_client.launch
```

Comment:
would this have a .xml postfix?

Reply:
Thanks for catching that, fixed in https://github.com/open-rmf/free_fleet/pull/107/commits/ce9169088436cf5fc64d558d8dafb74804d1ffe4